contracts/Remittance.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity 0.5.10;
+
+import "./SafeMath.sol";
+import "./Stoppable.sol";
+
+contract Remittance is Stoppable {
+    using SafeMath for uint;
+    
+    struct Order {
+        uint time;
+        address emitter;
+        uint amount;
+    }
+    
+    uint constant TX_FEES = 2000;
+
+    mapping(bytes32 => Order) public transactions;
+    mapping(address => uint) public fees;
+    
+    event LogNewTransaction(address indexed emitter, address exchange, uint amount);
```

Comment:
`LogNewOrder`?

Reply:
I think all your events should mention the hashed OTP. After all, that's the only thing that helps us follow the lifecycle of an order.

Reply:
Agree. All excepting the withdrawFees method function :)